bitnami/thanos/Chart.yaml

```diff
@@ -28,4 +28,4 @@ name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
   - https://thanos.io
-version: 3.10.1
+version: 3.10.4
```

Comment:
I think we should bump the minor version. You're including a new parameter, and modifying the expected format of another one.

Reply:
Done